src/lib/y2partitioner/clients/main.rb

```diff
@@ -27,7 +28,10 @@ class Main
       def self.run
         textdomain "storage"
 
+        probed = Y2Storage::StorageManager.instance.y2storage_probed
         staging = Y2Storage::StorageManager.instance.y2storage_staging
+        DeviceGraphs.instance.original = probed
+        DeviceGraphs.instance.current = staging
```

Comment:
well, this is just quick cherry pick from edit pull request. I expect that there they will init it in whatever way they prefer and what is needed for their functionality